### PR TITLE
fix(csv-parser-price): handle UTF-8 with BOM encoding while price import

### PR DESCRIPTION
**Pull request by VineetKumarKushwaha**
#### UTF-8 with BOM encoding while doing price import

Fixes: #1534

### CODE REVIEW

File: packages/csv-parser-price/src/main.js [hunk 1]
```diff
@@ -55,13 +56,14 @@ export default class CsvParserPrice {
 
     highland(input)
       // Parse CSV return each row as object
+      .through(stripBom())
```
Comment (daern91) [inline packages/csv-parser-price/src/main.js:59]: awesome! 👍

File: packages/csv-parser-price/src/main.js [hunk 2]
```diff
@@ -55,13 +56,14 @@ export default class CsvParserPrice {
...
       .through(
         csv({
           separator: this.delimiter,
           strict: true,
         })
       )
-      .map((a) => {
+      .map(a => {
```
Comment (daern91) [inline packages/csv-parser-price/src/main.js:66]: hmm, these formatting changes shouldn't happen. Could you please run `yarn format` in the root package folder to undo these changes? and maybe `yarn install` if you're running some old dependencies locally?

Comment (VineetKumarKushwaha) [reply packages/csv-parser-price/src/main.js:66]: It happens automatically. Sure will update.

Comment (VineetKumarKushwaha) [reply packages/csv-parser-price/src/main.js]: done
